gen3users/user_security.py

```diff
@@ -184,6 +208,21 @@ def _receive_input(anything_found):
     return
 
 
+def _f(subrsrc, phsid, locs):
+    for i in range(len(subrsrc)):
+        if subrsrc[i]["name"] == phsid:
+            return True, locs
+        try:
+            subrsrc[i]["subresources"]
+            locs.append(subrsrc[i]["name"])
+            subrsrc, locs = _f(subrsrc[i]["subresources"], phsid, locs)
+            if subrsrc == True:
+                return subrsrc, locs
+        except KeyError:
```

Comment:
Why would we pass on a keyerror here? Generally just passing on an error is a bad practice unless there's a really good reason to.